### ISSUE TITLE
The accuracy is different from the paper.

### ISSUE THREAD

**Issue by laiguoji** [prlz77/attend-and-rectify#3]
hi, i use your original code(it is wrn_28_10_ad=0_aw=0)  to train cifar10, and get the best val accuracy is 95.99%, but when i use the attention version(the params are the same as ./log/wrn_28_10_ad=3_aw=4_err=3.45), the best val acc is 95.88%.  it is even worse than the original code(without attention),  could you explain it?

**Comment by prlz77** [comment]
This is weird, I just reproduced them for another issue. Could you share your log and executing command?

**Comment by laiguoji** [comment]
Yes, could you please tell me your email? i will share my log with you through email.

**Comment by prlz77** [comment]
pau dot rodri 1 at gmail dot com, you can attach them here otherwise

**Comment by prlz77** [comment]
Looking at your log, I would say that one of the main differences is the batch size. Since you multiply it for the number of gpus, you should modify the learning rate accordingly. I would suggest you train without the multiplier: batch size 128, lr 0.1. Since the pytorch versions are different I would suggest you also try the previous version. Basically, try to keep as close as possible to the original setup. Also try with 5 different random seeds, you should get a number closer to the results in the paper in the median.

**Comment by laiguoji** [comment]
Thanks，i train your code without the multiplier: batch size 128, lr 0.1.  But among the 3 runs, the best val acc is 96.07%. I don't know why it is not as good as the result of the paper.

**Comment by prlz77** [comment]
Does it happen with all the other setups?

**Comment by laiguoji** [comment]
I also use the setups as your log(./log/wrn_40_10_ad=3_aw=4_err=3.45), the best val acc is only 96.03%.

**Comment by prlz77** [comment]
Weird, I'll try to run it again with the new version of pytorch. Meanwhile it would be interesting to see what results you get with the dropout versions. I reproduced these ones recently for another issue and there was no problem, the numbers where as in the paper.

**Comment by laiguoji** [comment]
Thank you for your reply, but i have not run the dropout versions yet.

**Comment by Fridaycoder** [comment]
Hi, @prlz77 I have the same problem, I can't get the same accuracy as in the paper, what's worse, I run train_cifar10_warn.sh with the default settings, the best accuracy is 95.5%. The biggest difference is that I use PyTorch 1.0.1, is that what really matter?

**Comment by prlz77** [comment]
@Fridaycoder it could be. I'll try to run again with pytorch 1.3.1
